package.json

```diff
@@ -46,9 +46,9 @@
   "license": "Apache-2.0",
   "homepage": "https://github.com/asyncapi/generator",
   "dependencies": {
-    "@asyncapi/openapi-schema-parser": "^1.0.0",
-    "@asyncapi/parser": "^0.18.3",
-    "@asyncapi/raml-dt-schema-parser": "^1.0.4",
+    "@asyncapi/openapi-schema-parser": "https://github.com/derberg/openapi-schema-parser#new-reg",
+    "@asyncapi/parser": "https://github.com/derberg/parser-js#mime-types",
+    "@asyncapi/raml-dt-schema-parser": "https://github.com/derberg/raml-dt-schema-parser#new-reg",
```

Comment:
I'd remove these 3 changes or make it point to the released version before you merged this PR. Feel free to bump RAML and OpenAPI schema parsers to 2.0.0.

Reply:
I will definitely not merge it, I just changed it this way to point to my changes and have passing tests.